### PR TITLE
Add shorewall IPv6 support

### PR DESCRIPTION
**Pull request by roedie**
Small patch which allow fail2ban to use shorewall for IPv6 bans.

### CODE REVIEW

File: config/action.d/shorewall.conf [hunk 1]
```diff
@@ -50,10 +52,33 @@ actionban = shorewall <blocktype> <ip>
...
+
+# Option:  actionunban
+# Notes.:  command executed when unbanning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    See jail.conf(5) man page
+# Values:  CMD
+#
+actionunban = shorewall6 allow <ip>
+
+# Option:  blocktype
```
Comment (sebres) [inline config/action.d/shorewall.conf]: Following one is not needed here (only changed values for IPv6 are necessary in the 2nd init-section)

File: config/action.d/shorewall.conf [hunk 2]
```diff
@@ -50,10 +52,33 @@ actionban = shorewall <blocktype> <ip>
...
-[Init]
-
 # Option:  blocktype
 # Note:    This is what the action does with rules.
 #          See man page of shorewall for options that include drop, logdrop, reject, or logreject
 # Values:  STRING
 blocktype = reject
+
+[Init?family=inet6]
+
```
Comment (sebres) [inline config/action.d/shorewall.conf]: The test cases will fail, as long as `actionban` and `actionunban` belong to section definition.

So just create a new init parameter `family` (empty for ipv4 and `6` for ipv6):
```
[Definition]
actionban = shorewall<family> <blocktype> <ip>
actionunban = shorewall<family> allow <ip>

[Init]
family =

[Init?family=inet6]
family = 6
```

Comment (roedie) [inline config/action.d/shorewall.conf]: I did the changes. Hope this is more to your liking.

**Comment by sebres** [comment]
Merged, Thanks!

**Pull request merged**